devel/gradle:6.8

yzgyyang opened the pull request:
```
devel/gradle: Update to 6.8

While I was there, also switch WWW to https.

Approved by: lwhsu

lwhsu reviewed (approved):
I'm not sure how many ports depend on this to build and it might need an exp-run.

yzgyyang commented:
> I'm not sure how many ports depend on this to build and it might need an exp-run.

I don't believe an exp-run has been completed in the past: https://www.freshports.org/devel/gradle/

A quick (and inaccurate) grep also shows
```
➜  ports ag -l "devel/gradle" | wc -l
      13
```
, so I think we will be fine :)

Please reapprove since I updated WWW to https and changed the commit msg.
